microservices/first/main.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask, request, jsonify
```

Comment:
Can you add documentation for each of these endpoints at the top of this file? Our documentation is out of date currently and it would be nice to have it checked into version control before we get our wiki off the ground

Reply:
Okay can do

Reply:
Can't do this at the moment. Not enough time.

Reply:
```suggestion
# TODO: Add endpoint documentation to Flask app [#44](https://github.com/jakejack13/SPEED/issues/44)
from flask import Flask, request, jsonify
```